src/qt/intro.cpp

```diff
@@ -361,6 +416,11 @@ void Intro::UpdatePruneLabels(bool prune_checked)
         storageRequiresMsg = tr("Approximately %1 GB of data will be stored in this directory.");
     }
     ui->lblExplanation3->setVisible(prune_checked);
+    ui->pruneGB->setEnabled(prune_checked);
+    static constexpr uint64_t nPowTargetSpacing = 10 * 60;  // from chainparams, which we don't have at this stage
+    static constexpr uint32_t expected_block_data_size = 2250000;  // includes undo data
```

Comment:
Where does this constant comes from?

Reply:
Real-world block size limit + undo data. But not part of this PR anyway.